homeassistant/components/ipma/config_flow.py

```diff
@@ -22,14 +22,11 @@ async def async_step_user(self, user_input=None):
         self._errors = {}
 
         if user_input is not None:
-            if user_input[CONF_NAME] not in self.hass.config_entries.async_entries(
-                DOMAIN
-            ):
-                return self.async_create_entry(
-                    title=user_input[CONF_NAME], data=user_input
-                )
-
-            self._errors[CONF_NAME] = "name_exists"
+            unique_id = f"{user_input[CONF_LATITUDE]}, {user_input[CONF_LONGITUDE]}"
+            await self.async_set_unique_id(unique_id)
+            self._abort_if_unique_id_configured()
```

Comment:
This won't work for existing config entries. How about passing latitude + longitude to `self._async_abort_entries_match` instead?